superchain/configs/mainnet/arena-z.toml

```diff
@@ -1,21 +1,22 @@
 name = "arena-z"
-chain_id = 7897
 public_rpc = "https://rpc.arena-z.gg"
 sequencer_rpc = "https://rpc.arena-z.gg"
 explorer = "https://explorer.arena-z.gg"
 superchain_level = 0
 governed_by_optimism = true
-standard_chain_candidate = true # This is a temporary field which causes most of the standard validation checks to run on this chain
+data_availability_type = "eth-da"
+chain_id = 7897
```

Comment:
From memory we may have wanted the chain_id nearer the top of the file (for easier human readability). I don't feel strongly but wanted to flag for @tessr's input.

Reply:
Can update this later.